rest/answering-machine-detection/outgoing-call/outgoing-call-1.5.x.rb

```diff
@@ -0,0 +1,19 @@
+# Get twilio-ruby from twilio.com/docs/ruby/install
+require 'twilio-ruby'
+
+# Get your Account SID and Auth Token from twilio.com/console
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+
+# Initialize Twilio Client
+@client = Twilio::REST::Client.new(account_sid, auth_token)
```

Comment:
@joliveros, this is a subtle issue that we might want to discuss. `@` is used for instance variables. Most of the examples we have here are not using a class, so making them instance variables is unnecessary.

The following contrived example tries to illustrate my point.
```ruby
class Example
  def initialize(client)
    @client = client
  # ^ In this example, the usage of an instance variable makes sense.
  end

  def make_call
    call = @client.api.calls('CA42ed11f93dc08b952027ffbc406d0868')
    # ...
  end
end
```

What do you think about removing the prefix `@` from `@client` for the new snippets starting now?

Reply:
I agree. I plead guilty to copy and paste :smile:

Reply:
Thank you!